src/Microsoft.AspNet.Mvc.Core/Infrastructure/DefaultActionSelector.cs

```diff
@@ -87,8 +88,7 @@ public Task<ActionDescriptor> SelectAsync(RouteContext context)
                     Environment.NewLine,
                     finalMatches.Select(a => a.DisplayName));
 
-                _logger.LogError("Request matched multiple actions resulting in ambiguity. " +
-                    "Matching actions: {AmbiguousActions}", actionNames);
+                _logger.AmbiguousActions(actionNames);
```

Comment:
I've actually made a bunch of conflicting changes in this file, you can revert all of these.